pkgs/tools/system/proot/default.nix

```diff
@@ -19,8 +24,15 @@ stdenv.mkDerivation {
     sed -i /CROSS_COMPILE/d src/GNUmakefile
   '';
 
-  buildInputs = [ talloc ] ++ lib.optional enablePython python;
-  nativeBuildInputs = [ docutils ] ++ lib.optional enablePython swig;
+  buildInputs = [ ncurses libarchive talloc ] ++ lib.optional enablePython python3;
+  nativeBuildInputs = [ pkg-config docutils ] ++ lib.optional enablePython swig;
+  patches = [
+    # without this patch the package does not build with python>3.7
+    (fetchpatch {
+      url = "https://github.com/proot-me/proot/pull/285.patch";
+      sha256= "1vncq36pr4v0h63fijga6zrwlsb0vb4pj25zxf1ni15ndxv63pxj";
```

Comment:
```suggestion
      url = "https://patch-diff.githubusercontent.com/raw/proot-me/proot/pull/285.patch";
      sha256 = "1vncq36pr4v0h63fijga6zrwlsb0vb4pj25zxf1ni15ndxv63pxj";
```

Reply:
@SuperSandro2000 
What is the difference?

Reply:
i have the same question. it does not help at all with the readability

Reply:
> What is the difference?

The URL redirects to the one I suggested. If you open the patch in your browser you will always end up on the second one. You don't know what GitHub does with the redirect in the future. Also PRs shouldn't be fetched.

Reply:
I would apply this, if not for the URL, but for `s/sha256=/sha256 =/`